Project/Src/Actions/MiscActions.cs

```diff
@@ -218,8 +218,7 @@ public override void Execute(TextArea textArea)
 
             if (textArea.Document.HighlightingStrategy.Properties.ContainsKey("LineComment"))
                 new ToggleLineComment().Execute(textArea);
-            else if (textArea.Document.HighlightingStrategy.Properties.ContainsKey("BlockCommentBegin") &&
-                     textArea.Document.HighlightingStrategy.Properties.ContainsKey("BlockCommentBegin"))
+            else if (textArea.Document.HighlightingStrategy.Properties.ContainsKey("BlockCommentBegin"))
```

Comment:
Possible this should have been `BlockCommentEnd`.

Reply:
I agree that End seem to be what was intended, but highlighting seem to work.
Do you have any way to try this?